src/web/assets/garnish/src/DragSort.js

```diff
@@ -55,12 +53,11 @@ export default Drag.extend(
      */
     getHelperTargetX: function () {
       if (this.settings.magnetStrength !== 1) {
-        this.getHelperTargetX._draggeeOffsetX = this.$draggee.offset().left;
+        const draggeeOffsetX = this.$draggee.offset().left;
+
         return (
-          this.getHelperTargetX._draggeeOffsetX +
-          (this.mouseX -
-            this.mouseOffsetX -
-            this.getHelperTargetX._draggeeOffsetX) /
+          draggeeOffsetX +
+          (this.mouseX - this.mouseOffsetX - draggeeOffsetX) /
```

Comment:
The reason for storing these values as properties on the function is that it drastically improves JS efficiency, because JS doesn’t have to garbage collect a bunch of variables.